src/example/constrained_poisson/constrained_poisson_multiplierFESpace.jl

```diff
@@ -0,0 +1,130 @@
+# ```
+# ```math
+#    \lamda_v \int_\Gamma u \, d\gamma = 2 \pi \lambda_v
+# ```
+# This problem can be assembled by introducing a MultiplierFESpace and combining it with the usual FESpace using a MultiFESpace. 
+# In this example, the manufactured solution $$u(x,y)=cos(4\pi(x^2 + y^2))$$ is used to test the method.
+
+# import necessary packages
+using Bcube
+using LinearAlgebra
+using SparseArrays
+using WriteVTK
+
+const outputpath = joinpath(@__DIR__, "../../../myout/constrained_poisson/")
+mkpath(outputpath)
+
+# Read 2D mesh
+mesh_path = joinpath(outputpath, "mesh.msh")
+gen_disk_mesh(mesh_path; lc = 3.2e-2)
+mesh = read_msh(mesh_path)
+
+# Choose degree and define function space, trial space and test space
+const degree = 2
+fs = FunctionSpace(:Lagrange, degree)
+U = TrialFESpace(fs, mesh)
+V = TestFESpace(U)
+
+# Define the multiplier trial space and corresponding test space
+Λᵤ = MultiplierFESpace(mesh, 1)
+Λᵥ = TestFESpace(Λᵤ)
+
+# The usual trial FE space and multiplier space are combined into a MultiFESpace
+TrialMultiFESpace = MultiFESpace(U, Λᵤ)
+TestMultiFESpace = MultiFESpace(V, Λᵥ)
+
+# Define volume and boundary measures
+dΩ = Measure(CellDomain(mesh), 2 * degree + 1)
+Γ = BoundaryFaceDomain(mesh, ("BORDER",))
+dΓ = Measure(Γ, 2 * degree + 1)
+
+# Define solution FE Function
+ϕ = FEFunction(U)
+
+# Define source term function (deduced from manufactured solution)
+f = PhysicalFunction(
+    x ->
+        64.0 * π^2 * (x[1]^2 + x[2]^2) * cos(4.0 * π * (x[1]^2 + x[2]^2)) +
+        16.0 * π * sin(4.0 * π * (x[1]^2 + x[2]^2)),
+)
+
+volume = sum(Bcube.compute(∫(PhysicalFunction(x -> 1.0))dΩ))
+
+# Define bilinear and linear forms
+function a((u, λᵤ), (v, λᵥ))
+    ∫(∇(u) ⋅ ∇(v))dΩ + ∫(side⁻(λᵤ) * side⁻(v))dΓ + ∫(side⁻(λᵥ) * side⁻(u))dΓ
+end
+
+# For the time being only functionals in the form of integrals can be assembled.
+# A temporary workaround is to put the multiplier in the integral and divide by the volume (the multiplier does not vary in space).
+l((v, λᵥ)) = ∫(f * v + 2.0 * π * λᵥ / volume)dΩ
+
+# Assemble to get matrices and vectors
+A = assemble_bilinear(a, TrialMultiFESpace, TestMultiFESpace)
+L = assemble_linear(l, TestMultiFESpace)
+
+# Solve problem
+sol = A \ L
+
+ϕ = FEFunction(TrialMultiFESpace)
+
+# Write solution and compare to analytical solution
+set_dof_values!(ϕ, sol)
+u, λ = ϕ
+
+println(" Value of Lagrange multiplier : ", λ.dofValues[1])
+
+ϕₑ = FEFunction(U)
+
+projection_l2!(ϕₑ, PhysicalFunction(x -> cos(4.0 * π * (x[1]^2 + x[2]^2))), mesh)
+
+Un = var_on_vertices(u, mesh)
+Ue = var_on_vertices(ϕₑ, mesh)
+mkpath(outputpath)
+dict_vars = Dict(
+    "Numerical Solution" => (Un, VTKPointData()),
+    "Analytical solution" => (Ue, VTKPointData()),
+)
+write_vtk(
+    outputpath * "result_constrained_poisson_equation_multiplierFESpace",
+    0,
+    0.0,
+    mesh,
+    dict_vars,
+)
+
+error = norm(Un .- Ue, Inf) / norm(Ue, Inf)
+println(" Linf Error : ", error)
```

Comment:
This part could be simplified thanks to the latest development (see for example [Poisson DG](https://github.com/bcube-project/BcubeTutorials.jl/blob/main/src/example/poisson_dg/poisson_dg.jl)).
A proposition here below :
```julia
u_ref = PhysicalFunction(x -> cos(4.0 * π * (x[1]^2 + x[2]^2)))
error = u_ref - u

vars = Dict("Numerical solution" => u, "Analytical solution" => u_ref, "error" => error)
filename = joinpath(outputpath, "output")
Bcube.write_vtk_lagrange(filename, vars, mesh, U)
    
l2(u) = sqrt(sum(Bcube.compute(∫(u ⋅ u)dΩ)))
el2 = l2(error)
tol = 1.e-10
@show el2
@assert el2 < tol
```